packages/react-storage/src/components/StorageBrowser/Views/Controls/Table.tsx

```diff
@@ -109,6 +109,10 @@ const LOCATION_DETAIL_VIEW_COLUMNS: Column<LocationItem>[] = [
     key: 'size' as keyof LocationItem,
     header: 'Size',
   },
+  {
+    key: 'download' as keyof LocationItem,
```

Comment:
These casts are problematic, what's the issue here

Reply:
hey Caleb, I think I originally added the columns for `lastModified` and `size`, so Emma might just be copying it. Originally though, it was complaining that the columns like `lastModified` did not match the `keyof` for the `LocationItem` type.

For instance:
```
Type '"lastModified"' is not assignable to type '"key" | "type"'.
```